diff --git a/net.sourceforge.Lifeograph.metainfo.xml.in b/net.sourceforge.Lifeograph.metainfo.xml.in
index ff60a6c7..5d5aa074 100644
--- a/net.sourceforge.Lifeograph.metainfo.xml.in
+++ b/net.sourceforge.Lifeograph.metainfo.xml.in
@@ -57,7 +57,7 @@
 
     <url type="homepage">http://lifeograph.sourceforge.net</url>
 
-    <content_rating type="oars-1.0" />
+    <content_rating type="oars-1.1" />
 
     <releases>
         <release version="3.0.0" date="2024-12-29">
